.github/workflows/test.yml

```diff
@@ -26,6 +41,14 @@ jobs:
         pip install tox
     - name: Execute tox
       env:
-        CI: "true"
+        PLATFORM: ${{ matrix.platform }}
+      # The "&&" syntax is supported on PowerShell and cmd
       run: |
         tox -e qa && tox -e docs && tox -e py-cov
+    - name: Report to codecov
+      # Ubuntu 18.04 came out of the box with 3.6, and LOTS of system are still running
+      # 18.04 happily, so we choose this as the 'canonical' code coverage testing.
+      # One day we'll have to revisit this and bump the version ...
+      if: always() && matrix.python-version == '3.6' && matrix.os == 'ubuntu-latest'
```

Comment:
Not sure if `always()` is needed or not... just in case it's needed, we add it here so codecov.io will receive reports of 100% coverage _and_ <100% ones.

Reply:
https://github.com/codecov/codecov-action is easier to configure IMHO. There is no need to check these matrix values anymore.

Reply:
Won't it upload the results of all 6 matrices, though? I don't know what will happen if we do that...

But it's definitely simpler and probably more reliable than making `bash` run a script pulled from the Internet.

I'll implement the change in a new commit.